handbook/marketing/index.md

```diff
@@ -5,8 +5,8 @@
 The marketing team holds a weekly meeting to review the topics below. [Internal notes](https://docs.google.com/document/d/1vX3vF6t8HP-7-K9giM85UKEoAhmhy-ifAvbbFDvmKwo/edit#) for each meeting should be maintained by the group (while adhering to the [company-wide communication guidelines](../communication/index.md))
 
 1. Metrics and performance against team quarterly [OKRs](../../company/okrs/index.md).
-1. Review of the prior week's initiatives.
-1. Plan for the coming week, and an update on monthly/quarterly initiatives.
+2. Review of the prior week's initiatives.
```

Comment:
No need to number these in the raw Markdown. They will show up with the correct numbers when rendered. The benefit of just using `1.` is that if you add or remove a step, you don't need to re-number all the others.